components/NewsItem/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import Upvote from '../Upvote'
 import PointsCount from '../PointsCount'
+import PropTypes from 'prop-types';
```

Comment:
I try and stick to the convention of absolute (module) imports above relative imports